website/docs/plugin/testing/acceptance-tests/plan-checks/output.mdx

```diff
@@ -90,13 +110,111 @@ func TestExpectKnownOutputValue_CheckPlan_Bool(t *testing.T) {
 }
 ```
 
-## Example using `plancheck.ExpectUnknownOutputValue`
+## `ExpectNullOutputValue` Plan Check
```

Comment:
Do we want to remove this documentation since we've recently decided to deprecate this? Or at least a note suggesting what to use instead.

Reply:
Have added a note for now.